pkg/broker/broker.go

```diff
@@ -459,14 +456,27 @@ func (a AnsibleBroker) Provision(instanceUUID uuid.UUID, req *ProvisionRequest,
 	}
 
 	context := &req.Context
-	parameters := &req.Parameters
+	parameters := req.Parameters
+
+	if req.PlanID == "" {
+		errMsg :=
+			"PlanID from provision request is blank. " +
+				"Provision requests must specify PlanIDs"
+		a.log.Error(errMsg)
+		return nil, errors.New(errMsg)
```

Comment:
This is going to print ```errMsg``` twice.  Here's where err gets passed to: https://github.com/openshift/ansible-service-broker/blob/master/pkg/broker/provision_job.go#L50. Same thing for all the custom error messages below.

Because we have the caller capturing and printing error everywhere, I think the only places it makes sense to print a custom error message is if you ```return nil, err```.  That way we print both the custom errMsg and the actually error.